sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -34,6 +34,8 @@ import org.apache.spark.sql.rapids.execution.TrampolineUtil
 import org.apache.spark.sql.types.{DateType, StructType, TimestampType}
 
 object GpuParquetFileFormat {
+  val PARQUET_WRITE_TIMESTAMP_CAST_TO_MILLIS = "com.nvidia.spark.rapids.parquet.write.castToMillis"
```

Comment:
Should this be in Rapids conf where all other confs are?

Reply:
I don't think we need a new config here.  We should just do what Spark is doing here and use the same SQLConf config.  That way if we add INT96 support it's straightforward.  Making this a boolean means we'll have to update it if/when INT96 or other types are supported.  Let's simplify and use the Spark conf key which precludes the need for a new conf key.